components/tooltip/tooltip-help.js

```diff
@@ -25,6 +25,10 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			 * @type {'top'|'bottom'|'left'|'right'}
 			 */
 			position: { type: String },
+			/**
+			 * @ignore
+			 */
+			showing: { type: Boolean, reflect: true },
```

Comment:
This should default to `false` in the constructor, otherwise it's `undefined` which is still `false`-y but not as nice. Don't you love JavaScript?!